packages/corvid-local-site/src/siteConverter.js

```diff
@@ -0,0 +1,354 @@
+/* eslint-disable no-useless-escape */
+
```

Comment:
maybe we should create something like: editorPaths (that will include all path data that is related to the editor) and that the site Converter won't be aware to any path (site/editor)

Reply:
i thought about it but didn't think it's really needed eventually